n2survey/lime/survey.py

```diff
@@ -588,3 +588,105 @@ def get_choices(self, question: str) -> dict:
             choices_dict = question_info.choices[0]
 
         return choices_dict
+
+    def rate_mental_health(self, question, condition):
+        """Calculate State Anxiety score based on responses to question.
+
+        Args:
+            question (str): Question ID to use for calculation
+            condition (str): Which kind of mental health condition to rate, "state", "trait", or "depression"
+
+        Returns:
+            df: Mental health condition ratings and classifications
+        """
```

Comment:
It is a bit hard to know what it does. Is there some description that we could reference? Or could you please add more in the description?

Reply:
Lea sent me part of the 2019 code that calculates these things. I re-wrote it in a more concise way. I can send you that code by email. There are also references in the 2019 report: 

1. K. Kroenke, R. L. Spitzer, J. B. W. William, and B. Löwe. The Patient Health Questionnaire somatic, anxiety,and depressive symptom scales: a systematic review. General Hospital Psychiatry, 32(4):345–359, 2010.
2. T. M. Marteau and H. Bekker. The development of a six-item short-form of the state scale of the spielberger state-trait
anxiety inventory (STAI). British Journal of Clinical Psychology, 31(3):301–306, 1992.

but I admit I didn't read them in full to carefully confirm everything in the 2019 code was correct; just decided to trust it.

Reply:
Could you please put these references in the docstring of the method. Just mention that these metrics are calculated according to the following sources.